docs/guide/upgrading-from-v0-15-to-v0-16.md

```diff
@@ -8,11 +8,6 @@ rather than a set of scripts.
 
 ## Breaking Changes
 
-### `download` is now a required callback for plugins
```

Comment:
Turns out I'd implemented things such that `download` is an optional callback. So this wasn't actually a change at all.